modules/Core/includes/endpoints/ServerInfoEndpoint.php

```diff
@@ -183,6 +183,18 @@ public function execute(Nameless2API $api) {
             $api->throwError(25, $api->getLanguage()->get('api', 'unable_to_update_server_info'), $e->getMessage());
         }
 
+        // Placeholder api
+        try {
+            foreach ($_POST['players'] as $uuid => $player) {
+                $user = new User($uuid, 'uuid');
```

Comment:
I think initialising a new User method might be overly resource heavy for a loop - imagine 100 people online each with a website user - could be wrong but might cause issues

Reply:
Okay, makes sense. I made a simple cache system as throughout the whole ServerInfoEndpoint it creates a new user obj 3x for each user online.